lib/ansible/modules/notification/rabbitmq_basic_publish.py

```diff
@@ -87,6 +86,12 @@
     file: 'path/to/logo.gif'
   delegate_to: localhost
 
+- name: RabbitMQ auto geneated queue
```

Comment:
s/generated/

Reply:
Hey @dch - thanks for the review.  I've fixed this now!